app/controllers/books_controller.rb

```diff
@@ -2,17 +2,26 @@ class BooksController < ApplicationController
   include Wor::Paginate
   before_action :authenticate_user!
   def index
-    @books = Book.all
-    render_paginated @books, each_serializer: CompleteBookSerializer
+    books = Book.all
+    OpenLibraryService.new(isbn: '0385472579').book_info
```

Comment:
What if this fails though? This is one of the differences between wor-requests and httparty, they handle failure differently. Read the documentation to see what will happen if the request fails and act accordingly (what should we tell the user? should it fail? how?

Reply:
Also, why are you calling OpenLibraryService with that particular isbn?